cspell.json

```diff
@@ -142,7 +142,7 @@
         "dups",
         "eaaccess",
         "EAS",
-        "elasticsearch",
+        "opensearch",
```

Comment:
since we are not removing elastic search completely, we should keep that one as well